### PR TITLE
Update routing tree location

### PR DESCRIPTION
**Pull request by stuartnelson3**
Routing tree editor is now only available as a
link from within the configuration docs for
alertmanager.

Code changes:
- Routing tree CSS is now in a CSS file
- The label set input now draws the route when
"enter" is pressed

@juliusv @fabxc

### CODE REVIEW

File: layouts/header.html [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
     <!-- Custom styles for this template -->
     <link href="/css/docs.css" rel="stylesheet">
+    <link href="/css/routing-tree-editor.css" rel="stylesheet">
```
Comment (fabxc) [inline layouts/header.html]: This is now used in every page. Don't know whether there's a decent way to do it differently with nanoc, so fine with me. 
Anyone bothered by that?

File: content/docs/alerting/configuration.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ While the command-line flags configure immutable system parameters, the
 configuration file defines inhibition rules, notification routing and
 notification receivers.
 
+An visual routing tree editor and label set matcher is available
+[here](/webtools/alerting/routing-tree-editor).
```
Comment (fabxc) [inline content/docs/alerting/configuration.md]: I'm generally not a fan of "here" links.

Maybe `The [visual editor]() can assist in building routing trees` or similar?

File: content/docs/alerting/configuration.md [hunk 3]
```diff
@@ -12,6 +12,9 @@ While the command-line flags configure immutable system parameters, the
 configuration file defines inhibition rules, notification routing and
 notification receivers.
 
+An visual routing tree editor and label set matcher is available
```
Comment (juliusv) [inline content/docs/alerting/configuration.md]: s/An/A/

File: layouts/header.html (hunk 1 above)
Comment (juliusv) [inline layouts/header.html]: We could of course add an `<%= if ... %>` check here if we really wanted. Like we do in https://github.com/prometheus/docs/blob/routing-tree-deux/layouts/header.html#L56.

But currently in docs.css we also have some styles that are only applicable for certain pages, so we're loading too much CSS already. It's not nice and we should optimize that, but I think it's small enough to still be ok for now.

File: content/docs/alerting/configuration.md (hunk 2 above)
Comment (stuartnelson3) [inline content/docs/alerting/configuration.md]: :+1:, fixes both comments

File: layouts/header.html (hunk 1 above)
Comment (stuartnelson3) [inline layouts/header.html]: Yeah, the total file custom css size is 2kb, so the concern is if these are overwriting any other styles. I'm pretty sure they aren't interfering with any other styles, though.

**Comment by juliusv** [comment]
:+1:

**Comment by stuartnelson3** [comment]
Will wait for agreement from @fabxc :)

**Comment by fabxc** [comment]
+1

**Pull request merged**